docs/medical-api/handling-data/schema/observation.mdx

```diff
@@ -4,41 +4,43 @@ description: "Schema documentation for the Observation table in the Metriport da
 group: "Schema"
 ---
 
-The observation table contains information on measurements other than lab tests e.g. blood pressure, height, and weight.
+The observation table contains information on observations.
+
+**Normalization Note:**
+Normalization of coding systems only occurs when the source coding system is one of: LOINC or SNOMED CT. Other coding systems will not have normalized codes.
```

Comment:
Worth saying what normalization means in this context. In relational DBs in general, this can be understood as creating tables to represent one to many and other types of relationships.

Maybe move the "Schema Documentation" intro from `data-analytics` into its own page and have a section about this? I think it'll be helpful, as "Schema Documentation" in the General after "Tuva" feels a bit out of place.